explore/src/main/scala/explore/config/SpectroscopyModesTable.scala

```diff
@@ -208,7 +202,17 @@ private object SpectroscopyModesTable extends TableHooks:
   private given Order[InstrumentRow#FPU]     = Order.by(_.toString)
   private given Order[BasicConfigAndItc]     = Order.by(_.configuration.configurationSummary)
 
-  private given Order[TimeSpan | Unit] = Order.by(_.toOption)
+  private def timeOrder(sort: Option[ColumnSort]): Order[Option[TimeSpan]] =
```

Comment:
Does this mechanism push undefined values to the bottom when the column is not sorted?

Reply:
Nope, the order is defined by the other columns